examples/torch/findAndAvoid/controllers/supervisor/supervisor.py

```diff
@@ -0,0 +1,229 @@
+import math
+
+import numpy as np
+
+import utilities as utils
+from deepbots.supervisor.controllers.supervisor_emitter_receiver import \
+    SupervisorCSV
+from deepbots.supervisor.wrappers.keyboard_printer import KeyboardPrinter
+from deepbots.supervisor.wrappers.tensorboard_wrapper import TensorboardLogger
+from models.default_networks import DefaultDDPG
+from models.networks import DDPG
+
+OBSERVATION_SPACE = 10
+ACTION_SPACE = 2
+
+DIST_SENSORS_MM = {'min': 0, 'max': 1023}
+EUCL_MM = {'min': 0, 'max': 1.5}
+ACTION_MM = {'min': -1, 'max': 1}
+ANGLE_MM = {'min': -math.pi, 'max': math.pi}
+
+
+class FindTargetSupervisor(SupervisorCSV):
+    def __init__(self, robot, target, observation_space):
+        super(FindTargetSupervisor, self).__init__(
+            emitter_name='emitter', receiver_name='receiver')
+        self.robot_name = robot
+        self.target_name = target
+        self.robot = self.supervisor.getFromDef(robot)
+        self.target = self.supervisor.getFromDef(target)
+        self.observation = [0 for i in range(observation_space)]
+        self.findThreshold = 0.12
+        self.steps = 0
+        self.steps_threshold = 6000
+        self.message = []
+        self.should_done = False
+
+    def get_observations(self):
+        message = super().handle_receiver()
```

Comment:
```suggestion
        message = self.handle_receiver()
```

It's generally better to call self instead of super, unless you explicitly want to call the super's method. If for example handle_receiver is overridden in FindTargetSupervisor class, this wouldn't have called it afaik, and would go straight for the super's handle_receiver.